compiler/pipes/parse-and-apply-phpdoc.cpp

```diff
@@ -266,7 +266,8 @@ class ParseAndApplyPhpDocForFunction {
       }
 
       case PhpDocType::kphp_extern_func_info: {
-        kphp_error(f_->is_extern(), "@kphp-extern-func-info used for regular function");
+        // This is temporary solution for generate-stub
```

Comment:
Please note here. I did it, because child classes with undeclared constructor get parent constructor